test/gtest/ucp/test_ucp_wireup.cc

```diff
@@ -401,6 +403,24 @@ bool test_ucp_wireup::ep_iface_has_caps(const entity& e, const std::string& tl,
     return false;
 }
 
+bool test_ucp_wireup::ep_lanes_has_caps(const entity& e, uint64_t caps)
```

Comment:
put this in ucp_test::entity::has_lane_with_caps(uint64_t caps)

Reply:
moved to ucp_test::entity::has_lane_with_caps